feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseController.java

```diff
@@ -0,0 +1,326 @@
+
+      // create a point located at Redlands, CA to be used as the viewpoint for the map
+      var point = new Point(-117.195800, 34.056295, SpatialReferences.getWgs84());
+      mapView.setViewpointCenterAsync(point, 10000);
+
+      // create a graphics overlay to display the input points
+      graphicsOverlay = new GraphicsOverlay();
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create a graphic to add the simple marker symbol in the graphics overlay
+      var simpleMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFF000000, 10);
+      var graphic = new Graphic();
+      graphic.setSymbol(simpleMarkerSymbol);
+      graphicsOverlay.getGraphics().add(graphic);
+
+      // keep track of the points added by the user
+      List<Point> inputs = new ArrayList<>();
+
+      // create a point from user input and add it to the geodatabase feature table
+      mapView.setOnMouseClicked(e -> {
+        if(createGeodatabaseButton.isDisabled() && !isTableWindowOpen) {
+          if (e.isStillSincePress() && e.getButton() == MouseButton.PRIMARY) {
+            // create 2D point from pointer location
+            var point2D = new Point2D(e.getX(), e.getY());
+
+            // create a map point from 2D point
+            Point mapPoint = mapView.screenToLocation(point2D);
+
+            // the map point should be normalized to the central meridian when wrapping around a map, so its value stays within the coordinate system of the map view
+            Point normalizedMapPoint = (Point) GeometryEngine.normalizeCentralMeridian(mapPoint);
+
+            // add a point where the user clicks on the map and update the inputs graphic geometry
+            inputs.add(normalizedMapPoint);
+            Multipoint inputsGeometry = new Multipoint(new PointCollection(inputs));
+            graphic.setGeometry(inputsGeometry);
+
+            if (inputs.size() > 0) {
+              // set up the feature attributes
+              Map<String, Object> featureAttributes = new HashMap<>(Map.of("collection_timestamp", Calendar.getInstance().getTime().toString()));
+              // create a new feature at the map point
+              var feature = geodatabaseFeatureTable.createFeature(featureAttributes, normalizedMapPoint);
+              // add the feature to the feature table
+              var addFeatureFuture = geodatabaseFeatureTable.addFeatureAsync(feature);
+              addFeatureFuture.addDoneListener(() -> {
+                try {
+                  addFeatureFuture.get();
+                  // update the total feature count on screen if feature was added successfully
+                  label.setText("Number of features added: " + geodatabaseFeatureTable.getTotalFeatureCount());
+
+                } catch (InterruptedException | ExecutionException ex) {
+                  throw new RuntimeException(ex);
+                }
+              });
+            }
+          }
+        }
+        else {
```

Comment:
java styling again, these are better readable on the same line

```suggestion
        } else {
```